apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -221,6 +221,7 @@ function main() {
 
```

Comment:
Ah, I see what you're saying — because we're calling it from new places, will those also work.

I've verified locally I'm able to load the right data in each tab of the Teacher Dashboard.  I'll create a follow up PR with UI test coverage!